students/elmar_m/lesson03/slicing_lab.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+
+'''
+file: slicing_lab.py
+elmar_m / 22e88@mailbox.org
+Lesson03: Slicing Lab Exercise
+Functions for slicing sequences.
+'''
+
+a_string = "this is a string"
+al_string = "this is an even longer string and it still goes on"
+a_tuple = (2, 54, 13, 12, 5, 32)
+al_tuple = (2, 54, 13, 12, 5, 32, 48, 102, 9, 11, 2056, 299)
+
+def exfl(a):
+    '''
+    return a given sequence with the first and last item exchanged
+    '''
+    return a[-1:] + a[1:-1] + a[:1]
+
+def onlyfl(a):
+    '''
+    return only the first and last item, remove everything in between 
+    '''
+    return a[:1] + a[-1:]
+
+def withoutfl4(a):
+    '''
+    remove the first 4 and the last 4 items of a given sequence and 
+    return everything in between
+    '''
+    return a[4:-4]
```

Comment:
this should be: "with the first 4 and the last 4 items removed, and then every other item in between."

so your `al_tuple` expected is: `(5, 48)`